src/schema/v2/artwork/index.ts

```diff
@@ -836,9 +836,32 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
 
           if (!vat_required) return true
 
-          const { vat_registered } = await partnerAllLoader(partner.id)
+          const { vat_status } = await partnerAllLoader(partner.id)
 
-          return vat_registered
+          return !!vat_status
+        },
+      },
+      vatExemptApprovalRequired: {
```

Comment:
After second thought, I updated the flag name to `vatExemptApprovalRequired` to make UI easier, given that's how UI would respect and render actions. We still need to add tests for this addition.

Reply:
Great, thanks for picking this up. I went ahead and wrote the tests for the new flag!

Reply:
Amazing! Thank you! 🙌